ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -57,7 +57,7 @@ final class Runner[T <: ReadWriteService, Extra](
           // initialize all configured participants
           _ <- Resource.sequence(config.participants.map { participantConfig =>
             val metricRegistry = factory.metricRegistry(participantConfig, config)
-            metricRegistry.registerAll(JvmServicePrefix, new JvmMetricSet)
+            metricRegistry.registerAll(JvmPrefix, new JvmMetricSet)
```

Comment:
Hmmmm… there's a "jvm" prefix added by `JvmMetricSet`, so these will end up in `daml.jvm.jvm`. Did you test this? (You can add `--metrics-reporter=console` to get metrics dumped every 10 seconds.)

If you add the prefix in `JvmMetricSet` instead, it'll work for Sandbox too.

Reply:
I'll double-check. Do we want that, i.e. to add the prefix for Sandbox too?

Reply:
Don't see why we shouldn't keep them aligned.